### PR TITLE
RST 6014 - fix: correctly displaying error messages when radio buttons are selected

### PR DESCRIPTION
**Pull request by lambley**

### CODE REVIEW

File: app/forms/respondent_form.rb [hunk 1]
```diff
@@ -95,4 +93,8 @@ def reset_work_address!
   def reload_addresses
     target.addresses.reload
   end
+
+  def validate_acas_reason_presence
```
Comment (garytaylor) [inline app/forms/respondent_form.rb:97]: is this used ?

File: spec/forms/respondent_form_spec.rb [hunk 2]
```diff
@@ -150,7 +155,7 @@
       describe 'when and ACAS number is given' do
         before { respondent_form.has_acas_number = 'true' }
 
-        it     { expect(respondent_form).not_to validate_presence_of(:no_acas_number_reason) }
```
Comment (garytaylor) [inline spec/forms/respondent_form_spec.rb:153]: why is it not important to prove that we are not validating no_acas_number_reason in this scenario ?

File: app/forms/respondent_form.rb (hunk 1 above)
Comment (lambley) [reply app/forms/respondent_form.rb]: no, should have removed that. removed now.

File: spec/forms/respondent_form_spec.rb (hunk 2 above)
Comment (lambley) [reply spec/forms/respondent_form_spec.rb:153]: would this be a better alternative? The test cases pass with this change:

```
describe 'when an ACAS number is given' do
  before { respondent_form.has_acas_number = 'true' }

  it     { expect(respondent_form).to validate_presence_of(:acas_early_conciliation_certificate_number) }
  it     { expect(respondent_form).to_not validate_presence_of(:no_acas_number_reason) }
end

describe 'when an ACAS number is given' do
  before { respondent_form.has_acas_number = 'false' }

  it     { expect(respondent_form).to validate_presence_of(:no_acas_number_reason) }
  it     { expect(respondent_form).to_not validate_presence_of(:acas_early_conciliation_certificate_number) }
end
```

Comment (garytaylor) [reply spec/forms/respondent_form_spec.rb:153]: yes that looks right

**Pull request merged**